CONTRIBUTING.rst

```diff
@@ -447,7 +447,7 @@ fetch and checkout a remote branch like this::
 
 .. _`Fork a Repo`: https://help.github.com/fork-a-repo/
 .. _`Rebasing merge commits in git`:
-    https://notes.envato.com/developers/rebasing-merge-commits-in-git/
+    https://web.archive.org/web/20150627054345/http://marketblog.envato.com/general/rebasing-merge-commits-in-git/
```

Comment:
Can probably replace this one with a live page, if anyone has a good suggestion for one

Reply:
I'm actually fine with this.